public/index.html

```diff
@@ -54,6 +54,33 @@
         <path fill="#FFF" d="M18 2L2 2 2 18 18 18z" transform="translate(5 5)"/>
         <path d="M12 13v2H5v-2h7zm3-4v2H5V9h10zm0-4v2H5V5h10z" transform="translate(5 5)"/>
       </symbol>
+      <symbol id="icon-menu" viewBox="0 0 32 32">
+        <title>menu</title>
+        <path d="M4 24h24v-2.667H4V24zm0-6.667h24v-2.666H4v2.666zm0 0h24v-2.666H4v2.666zM4 8v2.667h24V8H4z"/>
+      </symbol>
+      <symbol id="icon-account-owner" viewBox="0 0 32 32">
+        <title>account-owner</title>
+        <path fill="#FFF" d="M16 2.667C8.64 2.667 2.667 8.64 2.667 16S8.64 29.333 16 29.333 29.333 23.36 29.333 16 23.36 2.667 16 2.667z"/>
+        <path fill="#FF8415" d="M16 25.6c-3.333 0-6.28-1.707-8-4.293.04-2.654 5.333-4.107 8-4.107 2.653 0 7.96 1.453 8 4.107-1.72 2.586-4.667 4.293-8 4.293zM16 6.667c2.213 0 4 1.786 4 4 0 2.213-1.787 4-4 4s-4-1.787-4-4c0-2.214 1.787-4 4-4z"/>
+      </symbol>
+      <symbol id="icon-close-menu" viewBox="0 0 32 32">
+        <title>close</title>
+        <path d="M8.458 25.428l16.97-16.97-1.886-1.886-16.97 16.97 1.886 1.886zM6.572 8.458l16.97 16.97 1.886-1.886-16.97-16.97-1.886 1.886z"/>
+      </symbol>
+      <symbol id="icon-help" viewBox="0 0 20 20">
+        <title>help</title>
+        <path d="M0 0L20 0 20 20 0 20z"/>
```

Comment:
I think you'll get a better visual result if you delete this line that adds a background square.